### ISSUE TITLE
Look for alternative sources

### ISSUE THREAD

**Issue by seiteta** [datasets/cpi-us#4]
Need to check this is best data possible.

Current source: http://www.bls.gov/cpi/cpifiles/cpiai.txt

**Comment by AcckiyGerman** [comment]
Yes, this source is lost, and I could not find the proper source with historical data on their site on the first glance. Will come back later.

**Comment by ronwalf** [comment]
The data from the old cpiai.txt is a subset of what's found here (code CUSR0000SA0):
https://download.bls.gov/pub/time.series/cu/cu.data.1.AllItems

**Comment by olayway** [comment]
Resolved: the dataset now sources data from the BLS API v2 (series CUUR0000SA0, not seasonally adjusted), which provides the same data as the original flat file. No alternative source needed.
